### PR TITLE
Add gitignore

### PR DESCRIPTION
**Pull request by makostanyan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.idea/workspace.xml
```
Comment (arthur-zakaryan) [inline .gitignore:1]: If several files are from the same folder it would be good to exclude the entire folder.

Comment (makostanyan) [reply .gitignore:1]: Working on that. Thanks

Comment (arthur-zakaryan) [reply .gitignore]: Seems several files are in github which should have been ignored by gitignore. Pls take care of them.

Comment (makostanyan) [reply .gitignore]: Okay. I'll remove unnecessary files.

**Pull request merged**